moshi/japicmp/build.gradle.kts

```diff
@@ -31,14 +31,16 @@ val japicmp = tasks.register<JapicmpTask>("japicmp") {
     "com.squareup.moshi.internal.Util", // Internal.
     "com.squareup.moshi.StandardJsonAdapters", // Package-private
     "com.squareup.moshi.RecordJsonAdapter\$ComponentBinding", // Package-private
+    "com.squareup.moshi.ClassJsonAdapter\$FieldBinding", // Internal.
```

Comment:
Just exclude the whole class as it's package-private

Reply:
https://github.com/square/moshi/pull/1470/commits/442592115aa8faedd66afb33ea18b848b225aa63